sycl/test-e2e/Matrix/joint_matrix_bfloat16_colmajorA_colmajorB_impl.hpp

```diff
@@ -64,47 +64,29 @@ void matrix_multiply(big_matrix<T1, M, N> &C, big_matrix<T2, M, K> &A,
    }).wait();
 }
 
-static constexpr size_t MATRIX_M = TM * 2;
-static constexpr size_t MATRIX_N = TN * 2;
-static constexpr size_t MATRIX_K = TK * 2;
-bfloat16 A[MATRIX_K][MATRIX_M];
-bfloat16 B[MATRIX_N][MATRIX_K];
-float C[MATRIX_M][MATRIX_N];
-float D[MATRIX_M][MATRIX_N];
-
-void matrix_multiply_ref(int M, int N, int K) {
```

Comment:
this matrix_ref is different from the one above.
The one above uses matrix B that is VNNIed, so it uses two successive elements of B.
This one does not.
So I don't thin we can use same ref function for both.

Reply:
Not sure, I understand the comment. It seems to me the multiplication is equivalent in this test case.